packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -90,7 +116,19 @@ async function fundWallet(
   }
 
   // Options to pass to https.request
-  const httpOptions = getHTTPOptions(this, postBody, options?.faucetHost)
+  const httpOptions = getHTTPOptions(
+    this,
+    postBody,
+    options?.faucetHost,
+    options?.faucetPath,
+  )
+
+  // Currently, hooks testnet v2 has no way of funding a given wallet
+  if (httpOptions.hostname === 'hooks-testnet-v2.xrpl-labs.com' && wallet) {
+    throw new XRPLFaucetError(
+      'Cannot fund passed in wallet with hooks testnet v2',
```

Comment:
Ideally the error message is descriptive enough that the additional comment isn't necessary :) (Also added in a work around that was described to me)

```suggestion
  if (httpOptions.hostname === 'hooks-testnet-v2.xrpl-labs.com' && wallet) {
    throw new XRPLFaucetError(
            'Currently the Hooks Testnet v2 faucet has no way of funding a given wallet. If you need to do that, you can create a new funded account and have it send a payment transaction to the existing account.',
```